src/redux/actions/forecast/actions.test.js

```diff
@@ -0,0 +1,92 @@
+import querystring from 'querystring';
+import url from 'url';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import nock from 'nock';
+import axios from 'axios';
+import httpAdapter from 'axios/lib/adapters/http';
+
+import forecastActions from './index';
+import mock from './mock';
+
+const {
+  forecastRequest,
+  forecastSuccess,
+  forecastFailure,
+  receiveForecastAsync,
+} = forecastActions;
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('Forecast actions', () => {
+  it('forecast request', () => {
+    const action = forecastRequest();
```

Comment:
Not a bdd approach description. BDD descriptions should serve as a documentation understandable by not a technical person. it forecast request tells nothing. It should (necessary word for a bdd description) create an action of type...